api/typeDefs/user.js

```diff
@@ -209,8 +209,8 @@ export default gql`
     spent(when: String, from: String, to: String): Int
     referrals(when: String, from: String, to: String): Int
     streak: Int
-    gunStreak: Int
-    horseStreak: Int
```

Comment:
I believe removing these fields from the typeDef and resolvers will cause a brief outage on deployment.

Reply:
I forgot that the client/server can also be out of sync, not just server and db :grimacing: